runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/OrchestratorResponse.java

```diff
@@ -0,0 +1,48 @@
+package org.corfudb.protocols.wireprotocol.orchestrator;
+
+import io.netty.buffer.ByteBuf;
+import lombok.Getter;
+import org.corfudb.format.Types.OrchestratorResponseType;
+import org.corfudb.protocols.wireprotocol.ICorfuPayload;
+
+/**
+ * Created by Maithem on 10/25/17.
+ */
+
+public class OrchestratorResponse implements ICorfuPayload<OrchestratorResponse> {
+
+    @Getter
+    final public Response response;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)

Reply:
done